web/templates/user/_user_show_basedata.html

```diff
@@ -18,7 +18,7 @@
 
             {% call info_row("Name") %}
                 {% if current_user is privileged_for('user_change') %}
-                <a href="{{ url_for(".edit_user", user_id=user.id) }}">{{ user.name }} <span class="glyphicon glyphicon-edit"></span></a>
+                <span>{{ user.name }} </span><a href="{{ url_for(".edit_user", user_id=user.id) }}"><span class="glyphicon glyphicon-edit"></span></a>
```

Comment:
I think the `<span>`s aren't needed.

Reply:
Was unsure but apparently it does not make a difference so I removed the `<spans>`s.